### PR TITLE
Translate "CVE-2024-27280, CVE-2024-27281" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Actual diff is 1d15da64b721ec813187c1f58ab71981300f3720

### CODE REVIEW

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+tags: security
+lang: ko
+---
+
+버퍼 초과 읽기 취약점에 대한 보안 수정이 포함된 StringIO gem 버전 3.0.1.1과 3.0.1.2를 릴리스했습니다.
+이 취약점에는 CVE 식별자 [CVE-2024-27280](https://www.cve.org/CVERecord?id=CVE-2024-27280)이 할당되었습니다.
+
+## 세부 내용
+
+Ruby 3.0.x의 3.0.6까지와 3.1.x의 3.1.4까지 함께 배포된 StringIO 3.0.1에서 문제가 발견되었습니다.
```
Comment (yous) [inline ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md:16]: How about this?

```suggestion
Ruby 3.0.x부터 3.0.6까지, 3.1.x부터 3.1.4까지 함께 배포된 StringIO 3.0.1에서 문제가 발견되었습니다.
```

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+StringIO의 `ungetbyte`와 `ungetc` 메서드는 문자열의 끝을 넘어서도 읽을 수 있으며, 이어서 호출되는 `StringIO.gets`는 메모리의 값을 반환할 수 있습니다.
+
+이 취약점은 StringIO 3.0.3과 그 이후, 그리고 Ruby 3.2.x와 그 이후 버전에는 영향이 없습니다.
```
Comment (yous) [inline ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md:20]:
```suggestion
이 취약점은 StringIO 3.0.3과 그 이후, Ruby 3.2.x와 그 이후 버전에는 영향이 없습니다.
```

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## 권장 조치
+
+StringIO gem을 3.0.3이나 그 이상으로 업데이트하는 것이 좋습니다. 이전 Ruby 버전대에 포함된 버전과의 호환성을 보장하기 위해 다음과 같이 업데이트할 수 있습니다.
+
+* Ruby 3.0 사용자: `stringio` 3.0.1.1로 업데이트
+* Ruby 3.1 사용자: `stringio` 3.0.1.2로 업데이트
+
+주의: StringIO 3.0.1.2는 이 취약점뿐만 아니라 [[Bug #19389]](https://github.com/ruby/ruby/commit/1d24a931c458c93463da1d5885f33edef3677cc2)의 버그 수정을 포함하고 있습니다.
```
Comment (yous) [inline ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md:29]:
```suggestion
주의: StringIO 3.0.1.2는 이 취약점뿐만 아니라 [[Bug #19389]](https://github.com/ruby/ruby/commit/1d24a931c458c93463da1d5885f33edef3677cc2)의 버그 수정도 포함하고 있습니다.
```

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+`gem update stringio`를 사용하여 업데이트할 수 있습니다. bundler를 사용하는 경우 `gem "stringio", ">= 3.0.1.2"`을 `Gemfile`에 추가하세요.
```
Comment (yous) [inline ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md:31]:
```suggestion
`gem update stringio`를 사용하여 업데이트할 수 있습니다. bundler를 사용하는 경우 `gem "stringio", ">= 3.0.1.2"`를 `Gemfile`에 추가하세요.
```

File: ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+tags: security
+lang: ko
+---
+
+RCE 취약점에 대한 보안 수정이 포함된 RDoc gem 버전 6.3.4.1, 6.4.1.1, 6.5.1.1과 6.6.3.1을 릴리스했습니다.
+이 취약점에는 CVE 식별자 [CVE-2024-27281](https://www.cve.org/CVERecord?id=CVE-2024-27281)이 할당되었습니다.
+
+## 세부 내용
+
+Ruby 3.x의 3.3.0까지와 함께 배포된 RDoc 6.3.3부터 6.6.2에서 문제가 발견되었습니다.
```
Comment (yous) [inline ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md:16]:
```suggestion
Ruby 3.x부터 3.3.0까지 함께 배포된 RDoc 6.3.3부터 6.6.2까지 문제가 발견되었습니다.
```

File: ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+(RDoc 설정에 사용되는) `.rdoc_options`을 YAML 파일로 파싱할 때, 복원될 수 있는 클래스에 대한 제한이 없기 때문에, 객체 주입 및 유도된 원격 코드 실행이 가능합니다.
```
Comment (yous) [inline ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md:18]:
```suggestion
(RDoc 설정에 사용되는) `.rdoc_options`를 YAML 파일로 파싱할 때, 복원할 수 있는 클래스에 대한 제한이 없기 때문에, 객체 주입 및 그로 인한 원격 코드 실행이 가능합니다.
```

File: ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+문서 캐시를 읽어올 때도 조작된 캐시가 있을 경우, 객체 주입 및 유도된 원격 코드 실행이 가능합니다.
```
Comment (yous) [inline ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md:20]:
```suggestion
문서 캐시를 읽어올 때도 조작된 캐시가 있을 경우, 객체 주입 및 그로 인한 원격 코드 실행이 가능합니다.
```

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md]: 👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md]: 👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md]: 👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)

File: ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md (hunk 4 above)
Comment (riseshia) [reply ko/news/_posts/2024-03-21-buffer-overread-cve-2024-27280.md]: 👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)

File: ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md (hunk 5 above)
Comment (riseshia) [reply ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md]: 👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)

File: ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md (hunk 6 above)
Comment (riseshia) [reply ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md]: 👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)

File: ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md (hunk 7 above)
Comment (riseshia) [reply ko/news/_posts/2024-03-21-rce-rdoc-cve-2024-27281.md]: 👍 fixed [c63cc67](https://github.com/ruby/www.ruby-lang.org/pull/3205/commits/c63cc671dc7b27ccbf75ccb684b575dda66123cb)

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks!

**Pull request merged**